sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -241,6 +286,16 @@ export type KeyType = "EC" | "EC-HSM" | "RSA" | "RSA-HSM" | "oct";
 // @public
 export type KeyUsageType = "digitalSignature" | "nonRepudiation" | "keyEncipherment" | "dataEncipherment" | "keyAgreement" | "keyCertSign" | "cRLSign" | "encipherOnly" | "decipherOnly";
 
+// @public
+export interface KeyVaultCertificate {
+    cer?: Uint8Array;
+    contentType?: CertificateContentType;
+    readonly kid?: string;
+    readonly policy?: CertificatePolicy;
+    properties: CertificateProperties;
+    readonly sid?: string;
+}
+
 // @public
 export interface KeyVaultClientCreateCertificateOptionalParams extends coreHttp.RequestOptionsBase {
     certificateAttributes?: CertificateAttributes;
```

Comment:
We should remove CertificateAttributes and just move the properties which are actually settable onto this options class

Reply:
Couldn't do this today. Please check the updated description of this PR.

Reply:
This is part of the autogenerated code. We may want to make that a separate change from this PR.